.github/workflows/build.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: 1.46.0
+          toolchain: 1.49.0
```

Comment:
I guess `flume` requires `1.49` and up?  If so, could you please also update the version number in README.md ?